project/Dependencies.scala

```diff
@@ -29,7 +29,7 @@ object Dependencies {
   val commonsLang = "commons-lang" % "commons-lang" % "2.4"
   val commonsIo = "commons-io" % "commons-io" % "2.4"
   val cssParser = "net.sourceforge.cssparser" % "cssparser" % "0.9.18"
-  val contentApiClient = "com.gu" %% "content-api-client" % "9.5"
```

Comment:
If this is for the interactive atom, [isn't that in 10.3](https://github.com/guardian/content-api-scala-client/blob/96b8ad38ed3921e69992ee71eb0f14798ab973c1/CHANGELOG.md)?